client/packages.go

```diff
@@ -117,3 +118,27 @@ func (client *Client) Snap(name string) (*Snap, error) {
 
 	return pkg, nil
 }
+
+// RemoveSnap removes the snap with the given name
+func (client *Client) RemoveSnap(name string) error {
+	const errPrefix = "cannot remove snap"
+	var rsp response
+
+	path := fmt.Sprintf("/2.0/snaps/%s", name)
+	body := strings.NewReader(`{"action":"remove"}`)
+
+	if err := client.do("POST", path, body, &rsp); err != nil {
```

Comment:
Yeah, this code will eventually make it's way into a shared function but we all know the perils of premature optimisation ;)